cpu/stm32/periph/gpio_ll_irq.c

```diff
@@ -84,10 +84,8 @@
 #ifdef RCC_APB2ENR_SYSCFGCOMPEN
 #  define SYSFG_CLOCK           APB2
 #  define SYSFG_ENABLE_MASK     RCC_APB2ENR_SYSCFGCOMPEN
-#endif
-
-#ifdef  RCC_APBENR2_SYSCFGEN
-#  define SYSFG_ENABLE_MASK     RCC_APBENR2_SYSCFGEN
+#elif defined(RCC_APB2ENR_SYSCFGEN)
```

Comment:
This has to be an `#elif` instead because for STM32F0 both are defined:

```C
/* Old Bit definition maintained for legacy purpose */
#define  RCC_APB2ENR_SYSCFGEN                RCC_APB2ENR_SYSCFGCOMPEN        /*!< SYSCFG clock enable */
#define  RCC_APB2ENR_ADC1EN                  RCC_APB2ENR_ADCEN               /*!< ADC1 clock enable */
```